packages/editor/src/components/save-publish-panels/index.js

```diff
@@ -102,10 +108,47 @@ export default function SavePublishPanels( {
 	return (
 		<>
 			{ isEntitiesSavedStatesOpen && (
-				<EntitiesSavedStates close={ closeEntitiesSavedStates } />
+				<EntitieDialogWrapper close={ closeEntitiesSavedStates }>
+					<EntitiesSavedStates close={ closeEntitiesSavedStates } />
+				</EntitieDialogWrapper>
 			) }
 			<Slot bubblesVirtually />
 			{ ! isEntitiesSavedStatesOpen && unmountableContent }
 		</>
 	);
 }
+
+/**
+ * A wrapper component that renders a dialog for displaying entities.
+ *
+ * @param {Object}          props          The component's props.
+ * @param {React.ReactNode} props.children The content to be displayed inside the dialog.
+ * @param {Function}        props.close    A function to close the dialog.
+ *
+ * @return {React.Element} The rendered dialog element with children.
+ */
+export function EntitieDialogWrapper( { children, close } ) {
```

Comment:
this new wrapper should be used in other palces as well, for example for the save panel in the theme preview, see `EntitiesSavedStatesForPreview`

Reply:
The name of the component should make clear it is meant as a wrapper for `EntitiesSavedStates`.

Reply:
Noted. I only worked on save panel as If it was acceptable solution then probably can add to theme preview aswell.